api/src/main/resources/RegistrationEvent.hbm.xml

```diff
@@ -16,5 +16,10 @@
 			length="38" unique="true" />
 	</class>
 -->
+    <joined-subclass name="org.openmrs.module.registrationcore.api.mpi.common.MpiPatient"
```

Comment:
@djazayeri Actually it doesn't helped me to solve that problem. This is redundant code, which I forgot to remove.